modules/javafx.graphics/src/main/java/com/sun/javafx/css/BitSet.java

```diff
@@ -601,7 +601,7 @@ public void addListener(SetChangeListener<? super T> setChangeListener) {
     @Override
     public void removeListener(SetChangeListener<? super T> setChangeListener) {
         if (setChangeListener != null) {
```

Comment:
Even if not explicitly specified for `ObservableSet.removeListener(SetChangeListener)`, implementations generally reject `null` arguments by throwing NPE. This is the default behavior of `SetListenerHelper`.

Reply:
I could remove these checks in this PR, or as part of the larger `BitSet` clean-up PR.   The extra check doesn't break anything though, while this PR is mainly focused on fixing a long standing bug that everyone dealing with `ObservableSet<PseudoClass>` can encounter currently.